README.md

```diff
@@ -30,7 +30,7 @@ Edit `src/main/config/*.properties` files to specify parameters describing the e
 ```sh
 mvn package
 mkdir ${SECOR_INSTALL_DIR} # directory to place Secor binaries in.
-tar -zxvf target/secor-0.1-SNAPSHOT-bin.tar.gz -C ${SECOR_INSTALL_DIR}
+tar -zxvf target/secor-0.16-SNAPSHOT-bin.tar.gz -C ${SECOR_INSTALL_DIR}
```

Comment:
Thanks for the update.  Instead of replacing hard-coded 0.1 with 0.16 which is changing rapidly, can you use a token instead. e.g secor-<version>-bin.tar.gz, the current <version> can be retrieved from version field in pom.xml, e.g. 0.16-SNAPSHOT